content/source/assets/javascripts/alert-banner.js

```diff
@@ -0,0 +1,65 @@
+document.addEventListener('turbolinks:load', function () {
+  var qs = document.querySelector.bind(document)
+  var banner = qs('.g-alert-banner')
+  var closeButton = qs('.g-alert-banner .close')
+  // if cookie is set, don't show the component
+  if (getCookie(`banner_terraform_io`) === '1') {
+    banner.classList.remove('show')
+    return
+  }
```

Comment:
I wonder, would this be better as "if cookie _isn't_ set, _show_ the component"? 

The major difference is which loading method is best:
- showing the alert by default and removing it if cookie is set (as above)
- hiding the alert by default and showing it if cookie _isn't_ set

Thoughts?

Reply:
It's good to think this through - let's leave it as is. If someone has cookies disabled completely, then they wouldn't see the banner. Not at all a significant concern in this case, but I don't see a reason to change it here. Asking the question is solid though. 👊